autogpt_platform/backend/backend/data/block.py

```diff
@@ -282,14 +303,15 @@ def execute(self, input_data: BlockInput, **kwargs) -> BlockOutput:
 # ======================= Block Helper Functions ======================= #
 
 
-def get_blocks() -> dict[str, Block]:
+def get_blocks() -> dict[str, Type[Block]]:
     from backend.blocks import AVAILABLE_BLOCKS  # noqa: E402
 
     return AVAILABLE_BLOCKS
 
 
 async def initialize_blocks() -> None:
-    for block in get_blocks().values():
+    for cls in get_blocks().values():
+        block = cls()
```

Comment:
I assume `Block` constructor will never need any arguments but maybe add some static method or const in the `Block` class and avoid creating instances from generic `Type` altogether?

Reply:
What's the benefit of this?
Updated though.